modules/openapi-generator/src/test/java/org/openapitools/codegen/kotlin/spring/KotlinSpringServerCodegenTest.java

```diff
@@ -385,4 +385,33 @@ public void beanQualifiers() throws Exception {
                 "@RestController(\"org.openapitools.api.PingApiController\")"
         );
     }
+
+    @Test(description = "use Spring boot 3 & jakarta extension")
+    public void useSpringBoot3() throws Exception {
```

Comment:
I see several project examples inside samples/server/petstore but I don't see any example with jakarta, is that intended?

Reply:
these file changes are intended as i modified import style for existing ones. unit test is covering jakarta imports validation and not sure on what basis framework generate sample project for different types